quick_start.adoc

```diff
@@ -316,16 +316,16 @@ cm create authrealm --values <values.yaml>
 ----
 
 . Fill the template form
-+
+
```

Comment:
I think these plus signs were in there to make the numbering work properly.  Does the numbering still work if this is removed?

Reply:
And plus signs were for indentation to work properly too.

Reply:
yes check this page https://identitatem.github.io/idp-mgmt-docs/quick_start.html#getting-started